packages/orbit-components/src/Checkbox/Checkbox.ct.tsx

```diff
@@ -1,12 +1,157 @@
 import * as React from "react";
 import { test, expect } from "@playwright/experimental-ct-react";
+import { defaultTokens } from "@kiwicom/orbit-design-tokens";
 
-import CheckboxStory from "./Checkbox.ct-story";
+import { CheckboxStory, CheckboxWithTooltipStory } from "./Checkbox.ct-story";
 
 test.describe("visual Checkbox", () => {
+  const largeMobileBreakpoint = defaultTokens.breakpointLargeMobile;
+
   test("Checkbox", async ({ mount }) => {
     const component = await mount(<CheckboxStory />);
+    await expect(component).toHaveScreenshot();
+  });
+
+  test("Checkbox - disabled, active tooltip on desktop", async ({ mount, viewport }) => {
+    /**
+     * This command skips the test if the viewport width is bigger than largeMobile.
+     * Condition inside is based on the viewport width,
+     * which is specified for each of the device in playwright-ct.config.ts file.
+     * Using that we can distinguish mobileSmall, mobileMedium devices and larger devices.
+     */
+    test.skip(
+      viewport !== null && viewport.width < largeMobileBreakpoint,
+      "This feature is largeMobile, tablet and desktop-only",
+    );
+
+    const component = await mount(<CheckboxWithTooltipStory disabled />);
+
+    await component.getByRole("button").hover();
+    await expect(component).toHaveScreenshot();
```

Comment:
You said you added assertion to check the visibility of tooltip but you added it only to mobile scenarios, it's not necessary here? Why it's necessary for the mobile?

Reply:
ok, so let me explain how it works further.

we have two types of how the tooltip can be shown:
- "usual" design of tooltip - just classic tooltip component (shown on `desktop`, `tablet`, `largeMobile`)
![image](https://github.com/user-attachments/assets/ad19d916-53c0-43f1-8c67-8687365ab121)

- tooltip shown in the bottom part of the viewport, visible on click - this is implemented for `smallMobile` and `mediumMobile`

![image](https://github.com/user-attachments/assets/e1f8b05a-fa84-48e2-a6e3-306f4bb8a401)



**Why is this assertion set up only for `smallMobile` and `mediumMobile`?**
as I mentioned above, the tooltip for these display widths is shown after click, and sometimes the click is slower. Without that assertion (which waits for the visibility of respective component) the screenshot command could be called before the tooltip was really visible. In the case of display > `largeMobile`, we have tooltip shown on hover and there is not needed to wait as we are sure, that the tooltip is shown immediately.

And the reason why I replaced `browserName` condition by `viewport.width < largeMobileBreakpoint` is that `browserName` included `largeMobile` and `tablet` (these widths have a tooltip shown on hover). This way of displaying the tooltip causes there is attribute `role="tooltip"` twice and then the mentioned assertion fails, which is not the desired state (of course :D).

If there is something that's not still clear, feel free to ask.